### ISSUE TITLE
Missing tags?

### ISSUE THREAD

**Issue by durack1** [PCMDI/xml-cmor3-database#55]
@mauzey1 I was taking a look at https://github.com/PCMDI/xml-cmor3-database/tags and it seems we're missing `01.00.31`, `01.00.30`, `01.00.27`, `01.00.26`, `01.00.25` which most likely were generated, but just not tagged - right?

**Comment by mauzey1** [comment]
@durack1 Yes, those data requests were used to create CMIP6 tables but didn't get a release tag in xml-cmor3-database.  I'm not sure where 01.00.25 is since the commit history in both xml-cmor3-database and cmip6-cmor-tables for the tables makes it look like it went from 01.00.24 to 01.00.26.  @doutriaux1, do you know why this is the case?

**Comment by dnadeau4** [comment]
These tags were for the DREQ version.   The program should not be using any of the CMIP5 files anymore except for the "grids" which are not found in the DREQ table.
